fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/client/item/v1/FabricItemUpdateAnimationHandlers.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.item.v1;
+
+import java.util.Objects;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.item.ItemConvertible;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.registry.Registry;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.impl.client.ItemUpdateAnimationHandlerExtensions;
+
+/**
+ * Provides an API to register and retrieve {@link AnimationUpdateHandler}s for {@link ItemConvertible}s.
+ *
+ * <p>UpdateAnimationHandler defines behavior for whether or not the bobbing/reload animation should play after an ItemStack's NBT changes.
+ */
+@Environment(EnvType.CLIENT)
+public final class FabricItemUpdateAnimationHandlers {
```

Comment:
I recommend moving this to `AnimationUpdateHandler`.

Reply:
That is to have `register` and `get` as static functions inside `AnimationUpdateHandler`.